### DISCUSSION TITLE
Android 12

### DISCUSSION THREAD

**Discussion by FastenM** [AndreyPavlenko/Fermata#121] [General]
Is there any working fix for anyone who's on Android 12?

**Comment by AndreyPavlenko** [comment]
Unfortunately no. Try to downgrade AA to 6.9.

**Comment by 1Pedram** [comment]
> Unfortunately no. Try to downgrade AA to 6.9.

You mean no any upcoming solution at all? AA will not be able to work on outdated versions soon.

**Comment by AndreyPavlenko** [comment]
There is no other solution at the moment.

**Comment by technodrome** [comment]
> > Unfortunately no. Try to downgrade AA to 6.9.
> 
> You mean no any upcoming solution at all? AA will not be able to work on outdated versions soon.

Detach AA from Play store on Android 11. Either using Magisk or Titanium backup's cronjob. Use AA version 7.0.614134-release which does not nag about uptates once detached from PS.

**Comment by Alejandro2454** [comment]
Hi @Bohus kovac
¿Eres root?

**Comment by FastenM** [comment]
Someone told me that if you use wireless Android Auto Fermata works on Android 12 
(I can't verify his claims, I don't know what version of AA he has installed, and Im on Android 11)

Can someone on android 12 with wireless AA test it out?

**Comment by 1Pedram** [comment]
Tested on Android 12, latest AA, connecting Wirelessly: Fermata is in App list but not any action when tap on it and not working.

**Comment by fcaronte** [comment]
STEP FOR ONE TIME ONLY WORK ON ANDROID 12 (Maybe is bettern then nothing)
testen on android 12 with lineage os and samsung S21:
0- Install fermata apk using KingInstaller
1- Delete Android Auto data
2- Unlock AA Developer Mode 
3- Enable Unknown Source inside AA
IMPORTANT will work 1 time only and then when you stop using AA, from next time app will force close as usually and so YOU MUST REPEAT STEP FROM "1" TO HAVE ANOTHER WORKING SESSION. I try with airplane mode to check if is something server side but same block after first work try

**Comment by shrralis** [comment]
Is there at least a clue what is an obstacle for it's work on A12?

**Comment by fcaronte** [comment]
Practically is detect and blocked after the first time you use it

**Comment by Daleyboy69** [comment]
Really annoyed, paid for the premium version, and nothing works. Tried installing using Kingfisher,  tried clearing the cache and nothing. 

All that happens is I end up with a black screen on the head unit, and I have to unplug my phone to get the screen back.

**Comment by fcaronte** [comment]
I found the right version to use to have a normal working android auto with custom app. you need to uninstall android auto update and use version 6.9 but not a random one but the last one 6.9.613744, on my test phone i was on 6.9.613724 and wasn't work say me i need to update but with 6.9.613744 work perfectly for now @Daleyboy69

**Comment by FastenM** [comment]
@AndreyPavlenko for some reason the Fermata control app doesn't work but the main app does work, any idea how to fix? 

BTW Im already running Android 12 and yes I already downgraded AA to version 6.9.613744

**Reply by AndreyPavlenko** [reply]
What do you mean by it does not work?

**Reply by FastenM** [reply]
So when i open it in AA the screen is blank and it doesn't detect any media so it doesn't show up in the bottom where the media control is supposed to be 

And i also tested it by making sure something is activly playing in the background 

So basically every time i press home from the main app the media pauses.

**Reply by AndreyPavlenko** [reply]
When you start playing any media in Fermata Auto, does the control icon appears in the bottom?

**Reply by FastenM** [reply]
No, 


but i uninstalled the app and reinstalled and now its working.

**Comment by Salafi83** [comment]


**Reply by Salafi83** [reply]
I think Magisk roots your phone and I don't want to root my phone.

**Comment by fcaronte** [comment]
Tested few minutes ago and 6.9.613744 keep work correctly, you need to disable update from PlayStore and then keep attention to don't touch "update notification" will appear every time after you use android auto.
![IMG_20220122_130212_959](https://user-images.githubusercontent.com/26366403/150637757-edf40af3-1d54-4683-b7cf-a6c8cab2abf5.jpg)

**Comment by madala1819** [comment]
Is there any fix for fermata auto crashing on android 12

**Reply by zoe202212** [reply]
Downgrade android auto to version 6.9.613744
(Available at apkmirror.com)

**Comment by madala1819** [comment]
Hey haii thank you for your suggestion it's working now


Can you please guide me how to play Netflix and prime on android auto

Thank you

**Reply by AndreyPavlenko** [reply]
https://github.com/AndreyPavlenko/Fermata/discussions/131

**Comment by madala1819** [comment]
Thanks AndreyPavlenko/Fermata for your reply

But unfortunately I can't find installation instructions...could you please
guide me? Installation instructions or guidelines to get Netflix and prime
on android auto

Thanks in advance

**Reply by AndreyPavlenko** [reply]
Netflix does not work in Android WebView.

**Comment by madala1819** [comment]
Then what's that netflix player ? Which you had mentioned in last email?

And one more last question I want IPCARTV PLAY LISTS WHERE CAN I FIND THEM

**Reply by AndreyPavlenko** [reply]
I've sent a link to the corresponding discussion.

**Comment by madala1819** [comment]
I haven't got any link in the mail

**Reply by AndreyPavlenko** [reply]
You may find it here - https://github.com/AndreyPavlenko/Fermata/discussions

**Comment by madala1819** [comment]
Hey haii

How to play system files or videos on fermata auto

**Comment by ishanthap** [comment]
fermata 1.8.11 working fine with AA 6.9.613744, Android 12 on Samsung s10 thanks @AndreyPavlenko & @fcaronte

**Comment by AndreyPavlenko** [comment]
Seems, it works again with the latest AA release - https://www.apkmirror.com/apk/google-inc/android-auto/android-auto-7-2-6201-release/android-auto-7-2-620124-release-android-apk-download/

**Reply by fcaronte** [reply]
I can confirm is working correctly hope Google keep unlocked

**Reply by TSoft2022** [reply]
I confirm too, now it's working.

**Reply by FastenM** [reply]
For all saying it works on 7.2.620124 are any of you running android 12 or lower?

**Reply by TSoft2022** [reply]
I'm running Android 12 on a Samsung S21ultra

**Reply by FastenM** [reply]
Yep can confirm Fermata Auto works on AA version 7.2.620124 even on android 12 


@AndreyPavlenko

**Comment by madala1819** [comment]
Can anyone please suggest me how to play system media files on fermata auto

Thank you

**Reply by AndreyPavlenko** [reply]
What do you mean by "system media files"? Just add a folder with media files to the app.

**Comment by madala1819** [comment]
Ok, thanks.

**Comment by MickLesk** [comment]
Update Android Auto, then it works on Android 12.

https://android-auto.de.uptodown.com/android

**Reply by Asirelith** [reply]
Same, updated android Auto and it now works like a charm.

**Comment by EdmiltonJr** [comment]
I have a Galaxy S10 with Android 12, Android Auto 7.2.620124 and Fermata Auto 1.8.11 working normally since the 8th. Android Auto is the latest update from Play Store, official unmodified version. Fermata was installed by Kinginstaller.

![IMG_20220212_121024_866.jpg](https://user-images.githubusercontent.com/6454218/153716803-bd0524f9-bfa3-4a8e-9717-a6f73cc3da51.jpg)

![IMG_20220212_121028_218.jpg](https://user-images.githubusercontent.com/6454218/153716811-498144f5-2aca-4046-9d5c-40240522568a.jpg)

![IMG_20220212_121030_574.jpg](https://user-images.githubusercontent.com/6454218/153716820-73a855ad-44a4-411e-985d-43f988f7541d.jpg)
